### DISCUSSION TITLE
Official website's line spacing is too small and subtitle should bigger

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#7382] [General]
https://rustdesk.com/privacy.html
<hr>

![截圖2024-03-14 18 04 18](https://github.com/rustdesk/rustdesk/assets/25243894/d8319529-3361-4172-9fdf-25d401fae625)

**Comment by rustdesk** [comment]
Noted with thanks. :)

**Comment by rustdesk** [comment]
Updated
